python/py-line_profiler/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-line_profiler
+version             4.1.2
+revision            0
+supported_archs     arm64 x86_64
```

Comment:
is this needed? I don't see why it wouldn't work on another architecture? Please leave this to MacPorts' default unless you know for a fact (i.e., have verified this yourself or upstream is very clear about it) that it will only work on these architectures.

Reply:
Thank you. Fixed to default.